README.md

```diff
@@ -10,6 +10,82 @@ Utilities to help upgrading a ~250K line Rails app.
   * Rails tests
   * Jasmine specs
 
+## Pull Request Analyzer
```

Comment:
A lot of the formatting in this file isn't quite what we should have for Markdown.  Can we indent or use code fences (```) around the commands?